app/client/components/cancel/contributions/contributionsCancellationReasons.tsx

```diff
@@ -100,7 +104,14 @@ export const contributionsCancellationReasons: CancellationReason[] = [
         </p>
       </>
     ),
-    alternateFeedbackIntro
+    alternateFeedbackIntro,
+    shouldShow: (productDetail: ProductDetail) => {
+      const mainPlan = getMainPlan(productDetail.subscription);
+      if (!isPaidSubscriptionPlan(mainPlan)) {
+        return false;
+      }
```

Comment:
`isPaidSubscriptionPlan` should always be `true`, as this option is only rendered for a recurring contribution, but we need this for the type guard